File: 3830-find-closest-person/3830-find-closest-person.go
func f(x int) int {
    if x  < 0 {
        return -1 * (x)
    }
    return x
}
func findClosest(x int, y int, z int) int {
    d1, d2 := f(x - z), f(y - z)
    if d1 < d2 {
        return 1
    } else if d1 == d2 {
        return 0
    }
    return 2
}

File: 3830-find-closest-person/3830-find-closest-person.java
class Solution {
    public int findClosest(int x, int y, int z) {
        int d1 = Math.abs(x - z);
        int d2 = Math.abs(y - z);

        return (d1 >= d2) ? ((d1 == d2) ? 0 : 2) : 1;
    }
}